english/doug-adams/doug-adams.md

```diff
@@ -0,0 +1,8 @@
+Where did all the dolphins go?
+
+Dolphins are smart creatures, that much we know. However, it had never dawned upon us just how smart they actually were. The farewell message that they had sent prior to their departure was both enigmatic and ridiculous at the same time. _So long and thanks for all the fish_? Who would have thought... Who would have known?
+
+There are two people left in this stupendous galaxy who might know...
+
+[Arthur Dent](arthur-dent/arthur-dent.md)
+[Ford Prefect](ford-prefect/ford-prefect.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.